appchallenge/index.html

```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+<title>MCC</title>
+<meta charset="utf-8">
+<meta name="viewport" content="width-device-width, initial-scale=1">
+<link rel="stylesheet" href="http://code.jquery.com/mobile/1.4.5/jquery.mobile-1.4.5.min.css"/>
+<link rel="stylesheet" href="details.css"/>
+<script src="http://code.jquery.com/jquery-1.11.1.min.js"></script>
+<script src="http://code.jquery.com/mobile/1.4.5/jquery.mobile-1.4.5.min.js"></script>
+<!-- Ted Pysh
+	index.html
+	info2439
+	Thoendel
+	9/8/19 
+	-->
+
+
+</head>
+<body>
+<div data-role="page">
+	<div data-role="header">
+	<h1>MCC Sage Bistro</h1>
+	<img src="Sage-Bistro-Logo-2017-transparenent-small.png" id="logo">
+	</div>
+	<div data-role="header">
+	<a href="maps.html" class="ui-btn-right"> Find Us</a>
+	<a href="survey.html" class="ui-btn"> Take Survey</a>
+	<a href="reservation.html" class="ui-btn"> Reservation</a>
+	<a href="menu.html" class="ui-btn"> Menu</a>
```

Comment:
This is a lot of links for the header that are also included down below. Very functional, though!